travis.docker-compose.yml

```diff
@@ -0,0 +1,70 @@
+version: '2'
```

Comment:
There's a lot of duplication between our different docker-compose files, which can probably be dried out by relying on https://docs.docker.com/compose/extends/

Follow on is good for the moment.

Reply:
https://masschallenge.atlassian.net/browse/AC-5407